br/tests/config/pd.toml

```diff
@@ -5,3 +5,6 @@ enable-placement-rules = true
 cacert-path = "/tmp/backup_restore_test/certs/ca.pem"
 cert-path = "/tmp/backup_restore_test/certs/pd.pem"
 key-path = "/tmp/backup_restore_test/certs/pd.key"
+
+[keyspace]
+pre-alloc = ["keyspace1", "keyspace2"]
```

Comment:
will it affect normal tidb cluster where there is no keyspace setting?

Reply:
yeah it will, but non of the tests are using keyspace so they all fall back to default empty string.